.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+    tags: ["*"]
+
+jobs:
+  release:
+    # runs on main repo only
+    if: github.repository == 'akka/akka-management'
+    name: Release
+    # the release environment provides access to secrets required in the release process
+    # https://github.com/akka/akka-management/settings/environments
+    environment: release
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Checkout GitHub merge
+        if: github.event.pull_request
+        run: |-
+          git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
+          git checkout scratch
+
+      - name: Setup Scala with JDK 8
+        uses: olafurpg/setup-scala@v10
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v5
+
+      - name: Whitesource check policies and update
+        env:
+          WHITESOURCE_PASSWORD: ${{ secrets.WHITESOURCE_PASSWORD }}
+        run: sbt ";whitesourceCheckPolicies ;whitesourceUpdate"
+
+      - name: Publish artifacts for all Scala versions
+        env:
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+        run: sbt ci-release
+
+      - name: Publish API and reference documentation
+        env:
+          GUSTAV_KEY: ${{ secrets.GUSTAV_KEY }}
+          GUSTAV_PASSPHRASE: ${{ secrets.GUSTAV_PASSPHRASE }}
+        run: |+
+          eval "$(ssh-agent -s)"
+          echo $GUSTAV_KEY | base64 -di > .github/id_rsa
+          chmod 600 .github/id_rsa
+          ssh-keygen -p -P "$GUSTAV_PASSPHRASE" -N "" -f .github/id_rsa
+          ssh-add .github/id_rsa
+          sbt "++2.13.4 docs/publishRsync"
```

Comment:
Other workflows on this PR use 2.13.5 (e.g. https://github.com/akka/akka-management/pull/889/files#diff-cd0effce06f425599e96952f4d5f684641a683f46d9909e7c74a9f8b2f03ccd3R23).